ChangeLog

```diff
@@ -2,6 +2,10 @@ FUSE 2.9.8
 ==========
 
 * Fixed rename deadlock on FreeBSD.
+* Added a test of `seekdir` to test_syscalls.
```

Comment:
2.9.8 has already been released, so this needs a new entry ("Unreleased Changes")